src/Bonsai.Sleap/PredictCentroids.cs

```diff
@@ -71,6 +71,7 @@ private IObservable<CentroidCollection> Process(IObservable<IplImage[]> source)
                 TFSession.Runner runner = null;
                 var graph = TensorHelper.ImportModel(ModelFileName, out TFSession session);
                 var config = ConfigHelper.LoadTrainingConfig(TrainingConfig);
+                var ragged = graph["Identity_2"] != null;
```

Comment:
@shrivaths16 Just to confirm, in the older "ragged" versions of the centroid network, does `Identity_4` or `Identity_6` still exist?

I can also use the boolean complement:
var ragged = graph["Identity_4"] == null;

Reply:
Yes, you can use `Identity_4` for the boolean complement. 
Just so you have all the information,
`PredictPoses.cs` and `PredictCentroids.cs` uses the top-down model where we have up to `Identity_7` for the ragged version and only up to `Identity_3` for the unragged version.
`PredictPoseIdentities` used the top-down-id model where we have up to `Identity_6` for the ragged version and only up to `Identity_4` for the unragged version.